module.go

```diff
@@ -9,21 +9,26 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net"
+	"net/netip"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
 
 	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
 	"go.uber.org/zap"
 	"tailscale.com/tsnet"
 )
 
 func init() {
-	caddy.RegisterNetwork("tailscale", getPlainListener)
+	caddy.RegisterNetwork("tailscale", getTCPListener)
 	caddy.RegisterNetwork("tailscale+tls", getTLSListener)
+	caddy.RegisterNetwork("tailscale+udp", getUDPListener)
```

Comment:
tailscale/udp? Plus implies it's in addition too which is true for tls but not udp

Reply:
done